packages/labs/compiler/src/lib/template-transform.ts

```diff
@@ -65,6 +65,18 @@ interface TemplateInfo {
   variableName: ts.Identifier;
 }
 
+export interface CompileLitTemplateOptions {
+  /**
+   * Allows you to use `html` imported from modules other than lit.
+   * It is especially useful when re-exporting `html` tagged templates.
+   */
+  litHtmlSources?: string[];
```

Comment:
This is a great improvement! One open question is whether we should allow more than a static string here, e.g. maybe a regex?

How are relative module imports like the following handled?

```js
import {myHtml as html} from './reexport.js';
```

Reply:
> One open question is whether we should allow more than a static string here, e.g. maybe a regex?

I think, It probably be useful in the future. But now I can't think about a specific use case. Maybe for some bundlers, usually import statements are transformed and could be "useful" to tackle this transformation. I'm not really sure :C 

> How are relative module imports like the following handled?

For typescript './reexport.js' is the given importer specifier, so if you need to be compiled, You need to specify './reexport.js' (extension is important) as a litHtmlSource.

```js
import typescript from '@rollup/plugin-typescript';
import {compileLitTemplates} from '@lit-labs/compiler';

export default {
  // ...
  plugins: [
    typescript({
      transformers: {
        before: [
          compileLitTemplates({ litHtmlSources: './reexport.js' })
        ],
      },
    }),
    // other rollup plugins
  ],
};

``` 

Maybe some scenarios with relative import for the same re-exported html template can be suitable for regex... but usually, these cases are covered by import aliases.

What do yo think about use `createFilter` from [@rollup/plugin-utils](https://github.com/rollup/rollup-pluginutils) ?

Reply:
update: maybe we can support micromatch expressions and regex as createFilter, and not all rollup specific cases